opm/simulators/wells/GasLiftGroupInfo.cpp

```diff
@@ -294,7 +293,7 @@ checkDoGasLiftOptimization_(const std::string &well_name)
     auto itr = this->ecl_wells_.find(well_name);
     if (itr == this->ecl_wells_.end()) {
         // well_name is not present in the well_model's well container
-        displayDebugMessage_("Could find well ecl_wells. Skipping.", well_name);
+        displayDebugMessage_("Could not find well in ecl_wells. Skipping.", well_name);
         return false;
```

Comment:
I suggest we remove this debug message altogether. The well container only contains local wells i.e. this will currently spam the output and I don't think that the value of this message is significant.

Reply:
Yes I agree. Have updated this PR accordingly.

Reply:
Yes I agree. Have updated this PR accordingly.